cli/cmd/capture/capture.go

```diff
@@ -18,8 +21,8 @@ var capture = &cobra.Command{
 
 func init() {
 	cmd.Retina.AddCommand(capture)
-	configFlags = genericclioptions.NewConfigFlags(true)
-	configFlags.AddFlags(capture.PersistentFlags())
-	capture.PersistentFlags().StringVar(&name, "name", "", "The name of the Retina Capture")
+	opts.ConfigFlags = *genericclioptions.NewConfigFlags(true)
+	opts.AddFlags(capture.PersistentFlags())
+	capture.PersistentFlags().StringVar(opts.Name, "name", "", "The name of the Retina Capture")
```

Comment:
I know it's a small thing here, but this might be better as a method of `opts`. You can inject whatever the value of `capture.PersistentFlags()` is and let opts add Flags to it. It'd make it a little more modular by letting the opts say "I know what my flags should be".

Not a blocker, but just an idea.